test/runners/ocaml_spec.js

```diff
@@ -42,6 +42,10 @@ describe( 'ocaml runner', function(){
                 expect(buffer.stdout).to.contain("<DESCRIBE::>Person\n<IT::>.greet\n<PASSED::>");
                 done();
             })
-        })
+        });
+        it('should execute the included code examples', function(done){
+            runner.assertCodeExamples('ocaml');
```

Comment:
No need for the wrapped it. See the other languages for examples. I'm not sure if it will work inside of an `it` block.